File: ts/wam/7146.ts
export const WamGuestUpsellInteraction = 7146 as const

export interface GuestUpsellInteractionEvent {
  /** field 1, wire `guest_upsell_action` */
  guestUpsellAction?: GUESTUPSELLACTIONTYPE
  /** field 2, wire `guest_upsell_entry_point` */
  guestUpsellEntryPoint?: GUESTUPSELLENTRYPOINTTYPE
}

File: rust/wam/7146.rs
pub const WAM_GUEST_UPSELL_INTERACTION: u32 = 7146;

#[derive(Debug, Default)]
pub struct GuestUpsellInteractionEvent {
    /// field 1, wire `guest_upsell_action`
    pub guest_upsell_action: Option<GUESTUPSELLACTIONTYPE>,
    /// field 2, wire `guest_upsell_entry_point`
    pub guest_upsell_entry_point: Option<GUESTUPSELLENTRYPOINTTYPE>,
}